src/workspace-data/upload-data/UploadData.js

```diff
@@ -623,7 +623,8 @@ const MetadataUploadPanel = ({
       let otherRows = _.drop(1, rows);
 
       // Perform validation on the first row
-      if (!idColumn) {
+      // presence of '' in the first row indicates too many tab characters
+      if (!idColumn || _.indexOf('', headerRow) > -1) {
```

Comment:
clearly it works by demonstration, but this is confusing to me - how are we finding extra tabs by searching for (seemingly) nothing (`''`)?